lib/Bio/EnsEMBL/DataCheck/Checks/DisplayLabels.pm

```diff
@@ -0,0 +1,67 @@
+=head1 LICENSE
+
+Copyright [2018-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::DisplayLabels;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'DisplayLabels',
+  DESCRIPTION    => 'Check that certain tables have display_labels set',
+  GROUPS         => ['core', 'xref'],
+  DATACHECK_TYPE => 'critical',
```

Comment:
Might need to test this on current vertebrate dbs and if lots of things fail, might need to be advisory. We're testing for NULL values, which are allowed by the schema - if they really are so bad, why is the field NULLable in the first place?